owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -252,10 +350,9 @@ protected void setAccount(Account account, boolean savedAccount) {
     @Override
     protected void onAccountSet(boolean stateWasRecovered) {
         super.onAccountSet(mAccountWasRestored);
-        mReceiveExternalFilesViewModel = get(ReceiveExternalFilesViewModel.class);
+        suscribeViewModel();
```

Comment:
This should be called in `onCreate`. `onAccountSet` will be called always afterwards, and we could have some unexpected behaviour if we need some of the observers and code hasn't reached this method yet

Reply:
Done